intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,76 @@
+/* global $, Chart */
+$(document).ready(function() {
+    if (Object.keys(raw_data).length > 0) {
+        var chartElement = $("#members-chart");
+        chartElement.show();
+        var canvas = chartElement[0];
+        var parsedData = [];
+        $.each(raw_data, function(k, v) {
+            parsedData.push([moment(k, "YYYY-MM-DD"), v]);
+        });
+        parsedData = parsedData.sort(function(a, b) {
+            if (a[0] < b[0]) {
+                return -1;
+            }
+            if (a[0] > b[0]) {
+                return 1;
+            }
+            return 0;
+        });
+        parsedData = {
+            labels: $.map(parsedData, function(v) { return v[0]; }),
+            datasets: [
+                {
+                    label: "A Block",
+                    backgroundColor: "rgba(151,187,205,0.5)",
+                    data: $.map(parsedData, function(v) { return "A" in v[1] ? v[1].A : 0; })
+                },
+                {
+                    label: "B Block",
+                    backgroundColor: "rgba(205,187,151,0.5)",
+                    data: $.map(parsedData, function(v) { return "B" in v[1] ? v[1].B : 0; })
+                }
+            ]
+        };
+        Chart(canvas.getContext("2d"), {
+            type: "line",
+            data: parsedData,
+            options: {
+                maintainAspectRatio: true,
+                scales: {
+                    yAxes: [{
+                        type: "linear",
+                        scaleLabel: {
+                            display: true,
+                            labelString: "Signups"
+                        },
+                        ticks: {
+                            beginAtZero: true,
+                            suggestedMax: capacity
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["capacity" is not defined.](https://www.codacy.com/app/tjhsst-ion/ion/file/2688096538/issues/source?bid=3212528&fileBranchId=3299315#l49)